extensions/docbook-snippets/README.md

```diff
@@ -0,0 +1,29 @@
+# docbook-snippets README
+
+## Features
+
+The DocBook Snippets extension includes templates of both simple and complex DocBook structures.
+You can populate the templates by pressing by the initial DocBook markup letters. For example,
+pressing `li` results in `<listitem>insert content</listitem>`. The list of templates is not complete and will be extended with future releases.
+
+![Adding <itemizedlist/>](/extensions/docbook-snippets/media/itemizedlist.webm)
+
+## Requirements
+
+This extension has no dependencies.
+
+## Extension Settings
+
+This extension does not provide any settings yet.
+
+## Known Issues
+
+No issues yet.
```

Comment:
It's a bit dangerous to state "no issues yet" as the situation _will_ change in the near future. :wink: Maybe link to the issue tracker at GitHub? 

```suggestion
See our issue tracker at [openSUSE/suse-vscode-doc/issues](https://github.com/openSUSE/suse-vscode-doc/issues)
```